tests/testthat/test-get_custom_effort.R

```diff
@@ -1,33 +1,86 @@
 test_that("get_custom_effort returns error for invalid group_by value", {
-  expect_error(get_custom_effort(mica, group_by = "bad_value"))
+  expect_error(get_custom_effort(mica, group_by = "bad_value"),
+               regexp = paste0("Invalid value for group_by parameter: ", 
+                               "bad_value.\n",
+                               "Valid inputs are: NULL, day, week, month ", 
+                               "and year."),
+               fixed = TRUE
+  )
 })
 
 test_that("get_custom_effort returns error for start not a Date", {
   expect_error(get_custom_effort(mica, start = "2021-01-01"))
   # no datetime allowed
-  expect_error(get_custom_effort(mica, start = lubridate::as_datetime("2021-12-05 22:25:01 CET")))
+  expect_error(
+    get_custom_effort(
+      mica, 
+      start = lubridate::as_datetime("2021-12-05 22:25:01 CET")),
+    regexp = paste0("`start` must be `NULL` or an object of class Date. ",
+                    "Did you forget to convert a string to Date with ",
+                    "`as.Date()`?"),
+    fixed = TRUE
+  )
 })
 
 test_that("get_custom_effort returns error for end not a Date", {
   expect_error(get_custom_effort(mica, end = "2021-01-01"))
   # no datetime allowed
-  expect_error(get_custom_effort(mica, end = lubridate::as_datetime("2021-12-05 22:25:01 CET")))
+  expect_error(
+    get_custom_effort(mica,
+                      end = lubridate::as_datetime("2021-12-05 22:25:01 CET")),
+    regexp = paste0("`end` must be `NULL` or an object of class Date. ",
+                    "Did you forget to convert a string to Date with ",
+                    "`as.Date()`?"),
+    fixed = TRUE
+  )
 })
 
 test_that("get_custom_effort returns error if end earlier than start", {
-  expect_error(get_custom_effort(mica,
-    start = as.Date("2021-01-01"),
-    end = as.Date("1990-01-01")
-  ))
+  expect_error(
+    get_custom_effort(mica, 
+                      start = as.Date("2021-01-01"), 
+                      end = as.Date("1990-01-01")),
+    regexp = paste0("`end` value is set too early. `end` value must be not ", 
+                    "earlier than the start of the earliest deployment: ", 
```

Comment:
```suggestion
    get_custom_effort(mica,
                      start = as.Date("2021-01-01"),
                      end = as.Date("1990-01-01")),
    regexp = paste0("`end` value is set too early. `end` value must be not ",
                    "earlier than the start of the earliest deployment: ",
```
remove trailing whitespace